drivers/scheduler/k8s/specs/mysql-backup/px-mysql-storage.yaml

```diff
@@ -3,18 +3,12 @@ kind: Secret
 metadata:
   name: volume-secrets
 type: Opaque
-data:
-  mysql-secret: WW91IHNuZWFreSBsaXR0bGUgcGlnbGV0IQ==
 ---
 ##### Portworx persistent volume claim
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: mysql-data
-  annotations:
-    px/secret-name: volume-secrets
```

Comment:
When `volume-secrets` are not being used, better to remove entire secret object itself.

Reply:
done